manifests/init.pp

```diff
@@ -95,7 +101,9 @@
   String  $config_file            = $telegraf::params::config_file,
   String  $config_file_owner      = $telegraf::params::config_file_owner,
   String  $config_file_group      = $telegraf::params::config_file_group,
+  String  $config_file_mode       = $telegraf::params::config_file_mode,
   String  $config_folder          = $telegraf::params::config_folder,
+  String  $config_folder_mode     = $telegraf::params::config_folder_mode,
```

Comment:
same here. https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/filemode.pp. Also this type requires puppetlabs-stdlib 5.0.0 or newer. Please check that the metadata.json includes this version and not an older one.

Reply:
updated